src/ddmd/builtin.d

```diff
@@ -266,6 +297,24 @@ public extern (C++) void builtin_init()
     }
     // @safe @nogc pure nothrow long function(real)
     add_builtin("_D3std4math6rndtolFNaNbNiNfeZl", &eval_unimp);
+
+    // @safe @nogc pure nothrow T function(T, int)
+    add_builtin("_D4core4math5ldexpFNaNbNiNfeiZe", &eval_ldexp);
+    add_builtin("_D3std4math5ldexpFNaNbNiNfeiZe", &eval_ldexp);
+    add_builtin("_D3std4math5ldexpFNaNbNiNfdiZd", &eval_ldexp);
+    add_builtin("_D3std4math5ldexpFNaNbNiNffiZf", &eval_ldexp);
+
+    // @trusted @nogc pure nothrow bool function(T)
+    add_builtin("_D3std4math12__T5isNaNTeZ5isNaNFNaNbNiNeeZb", &eval_isnan);
+    add_builtin("_D3std4math12__T5isNaNTdZ5isNaNFNaNbNiNedZb", &eval_isnan);
+    add_builtin("_D3std4math12__T5isNaNTfZ5isNaNFNaNbNiNefZb", &eval_isnan);
+    add_builtin("_D3std4math18__T10isInfinityTeZ10isInfinityFNaNbNiNeeZb", &eval_isinfinity);
+    add_builtin("_D3std4math18__T10isInfinityTdZ10isInfinityFNaNbNiNedZb", &eval_isinfinity);
+    add_builtin("_D3std4math18__T10isInfinityTfZ10isInfinityFNaNbNiNefZb", &eval_isinfinity);
+    add_builtin("_D3std4math15__T8isFiniteTeZ8isFiniteFNaNbNiNeeZb", &eval_isfinite);
+    add_builtin("_D3std4math15__T8isFiniteTdZ8isFiniteFNaNbNiNedZb", &eval_isfinite);
+    add_builtin("_D3std4math15__T8isFiniteTfZ8isFiniteFNaNbNiNefZb", &eval_isfinite);
```

Comment:
From what I recall, they still are intrinsics, they were just given function bodies so you can take the address of them.